inst/examples/3_ReportingWorkflow.R

```diff
@@ -13,60 +13,85 @@ lData <- list(
     Source_QUERY = clindata::edc_queries,
     Source_ENROLL = clindata::rawplus_enroll,
     Source_SITE = clindata::ctms_site,
-    Source_STUDY = clindata::ctms_study
+    Source_STUDY = clindata::ctms_study 
 )
-ingest_wf <- MakeWorkflowList(strNames = "1_ingest")
-raw <- RunWorkflows(ingest_wf, lData)
 
-# Step 2 - Create Mapped Data - filter, aggregate and join raw data to create mapped data layer
-map_wf <- MakeWorkflowList(strNames = "2_map")
-mapped <- RunWorkflows(map_wf, raw$lRaw)
+# Step 0 - Data Ingestion - standardize tables/columns names 
+lRaw <- list(
+    Raw_SUBJ = Source_SUBJ,
```

Comment:
```suggestion
    Raw_SUBJ = lSource$Source_SUBJ,
```

Find and replace "Source_" with "lSource$Source_" for lines 21-43 ?